src/plugins/console/public/application/hooks/use_send_current_request/send_request.ts

```diff
@@ -127,25 +127,18 @@ export function sendRequestToES(args: EsRequestArgs): Promise<ESRequestResult[]>
         }
       } catch (error) {
         let value;
-        let contentType: string | null = '';
+        const { response, body } = error as IHttpFetchError;
+        const contentType = response?.headers.get('Content-Type') ?? 'unknown';
```

Comment:
nit: I'm not sure if we could use the string "unknown" here. Is there a specific reason to not keep an empty string if there is no content-type?

Reply:
`contentType` was [set](https://github.com/elastic/kibana/blob/0d9576fbce58399dc6b9b3218aa6e760d498dcee/src/plugins/console/public/application/hooks/use_send_current_request/send_request.ts#L27) to be  [BaseResponseType](https://github.com/elastic/kibana/blob/97f41c6b2adb2d4f81358b6c8986303ab54dc69f/src/plugins/console/public/types/common.ts#L26). But empty string makes more sense here. Thanks for the heads up